specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/definitions.json

```diff
@@ -7746,6 +7769,58 @@
       "additionalProperties": {
         "type": "string"
       }
+    },
+    "DocumentationContract": {
+      "properties": {
+        "properties": {
+          "properties": {
+            "title": {
+              "type": "string",
+              "description": "documentation title."
+            },
+            "content": {
+              "type": "string",
+              "description": "Markdown documentation content."
+            }
```

Comment:
Shouldn't these two be in 2nd level properties scope, and not in 3rd level properties? Based on your examples response, looks to be 2nd level.